at_secondary/at_secondary_server/test/sync_verb_test.dart

```diff
@@ -81,114 +76,7 @@ void main() async {
       expect(paramsMap['regex'], 'me');
     });
   });
-
-  group('A group of sync verb handler tests', () {
```

Comment:
why are these tests commented ?

Reply:
Earlier, sync response is added to the Response object. Now that we are directly writing to the socket, removed the test cases.